testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java

```diff
@@ -71,31 +71,68 @@ public class LayersTest {
      * @param unused The set of modules that are OK to not be provisioned when all layers are provisioned.
      * If more modules than this set are not provisioned, it means that we are missing some modules and
      * an error occurs.
-     * @throws Exception
+     * @throws Exception on failure
      */
+    @Deprecated
     public static void test(String root, Set<String> unreferenced, Set<String> unused) throws Exception {
+        testBootInstallations(root);
```

Comment:
This should be a call to the existing testExecutions(String) as the new testBootInstallations does the same thing.

Reply:
Sorry, I didn't realize that